lms/djangoapps/experiments/flags.py

```diff
@@ -209,16 +210,17 @@ def get_bucket(self, course_key=None, track=True):
             if bucket_flag.is_enabled(course_key):
                 bucket = i
                 break
-        else:
-            bucket = stable_bucketing_hash_group(
-                bucketing_group_name, self.num_buckets, user.username
-            )
+            else:
+                bucket = stable_bucketing_hash_group(
+                    bucketing_group_name, self.num_buckets, user
+                )
```

Comment:
This was just a floating `else` 😱

It was inline with the preceding `for`, so I made the assumption that it goes here?

Reply:
https://tutorialdeep.com/knowhow/else-with-for-loop-python/

Reply:
😱 x2